system/jlib/jflz.cpp

```diff
@@ -676,6 +682,9 @@ class jlib_decl CFastLZCompressor : public CInterface, public ICompressor
 
     virtual void open(void *buf,size32_t max)
     {
+        wrmax = max / 4;
+        if (wrmax < 256)
+            wrmax = max;
```

Comment:
I want it to be one fourth of max but not too small, in which case it should be at most max.